### PR TITLE
LOC CHECKIN | dotnet/sdk master | 20190114

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/Tasks/Common/Resources/xlf/Strings.cs.xlf [hunk 1]
```diff
@@ -109,22 +108,20 @@
       </trans-unit>
       <trans-unit id="NoRuntimePackAvailable">
         <source>NETSDK1082: There was no runtime pack for {0} available for the specified RuntimeIdentifier '{1}'.</source>
-        <target state="new">NETSDK1082: There was no runtime pack for {0} available for the specified RuntimeIdentifier '{1}'.</target>
```
Comment (nguerrera) [inline src/Tasks/Common/Resources/xlf/Strings.cs.xlf]: Some of these have had the target node removed entirely. 

This is causing xliff-tasks to crash.

FYI, @tmeschter.

I'm doing a fixup, but we should understand the root cause.

**Review by nguerrera — APPROVED** [review]

File: src/Tasks/Common/Resources/xlf/Strings.cs.xlf (hunk 1 above)
Comment (janbaltus) [reply src/Tasks/Common/Resources/xlf/Strings.cs.xlf]: Hi Nick, the string is not localized in the platform yet. (new string) and seems tool is stripping target tag when no translation exist. We will log this as request to be fixed by the tool owners.

Comment (nguerrera) [reply src/Tasks/Common/Resources/xlf/Strings.cs.xlf]: Thanks, yes. It should leave the target alone in this case.

Comment (nguerrera) [reply src/Tasks/Common/Resources/xlf/Strings.cs.xlf]: This was handled better in the past, by the way, so this seems to be a new problem.

Long ago, I requested changes to be made against the branch as it was when the handoff occurred, and then let git identify conflicts. We could then resolve by accepting in favor of the handback and letting the tool add back the new strings. That would address this among other things.

**Pull request merged**